sdk/communication/communication-sms/karma.conf.js

```diff
@@ -62,7 +62,8 @@ module.exports = function(config) {
       "TEST_MODE",
       "AZURE_CLIENT_ID",
       "AZURE_CLIENT_SECRET",
-      "AZURE_TENANT_ID"
+      "AZURE_TENANT_ID",
+      "COMMUNICATION_SKIP_INT_SMS_TEST"
```

Comment:
nit: same comment as above. add here https://github.com/Azure/azure-sdk-for-js/blob/95467bc594aa4196cb04b5b1bbf58599ef14d97e/sdk/communication/communication-sms/test/public/utils/recordedClient.ts#L16